### DISCUSSION TITLE
Is it possible to move to another timeline while resizing?

### DISCUSSION THREAD

**Discussion by oleh-andriiv** [samuelarbibe/dnd-timeline#49] [Q&A] [answered]
Let's imagine you have time on the X-axis such as 00-24 and dates on Y axis. Let's say I have an event starting from 8PM till 11 AM (so it is the same timeline within a single day), now I'd like to reschedule it and by resizing, extend this event to the second timeline (second day), till 8AM. So by moving down mouse cursor while resizing, it would jump to the second timeline, that way it would be possible to do it under single resize event. 

You can look into [this](https://demo.mobiscroll.com/fullscreen/timeline/monthly-timetable-vertical-days-horizontal-times#theme=ios&themeVariant=light) example, should be easier to understand.

**Comment by samuelarbibe** [comment]
By "moving to a second timeline", I assume you want to adjust the timeline range **while** dragging or resizing.  
I suggest you to modify the "range" prop while dragging or resizing, as mentioned in this [discussion](https://github.com/samuelarbibe/dnd-timeline/discussions/48).  
You may be able to find a shared solution!

**Reply by oleh-andriiv** [reply]
Actually I meant moving down by Y axis, while resizing, just like on the example link I've attached, where you can "jump" from one day to the second one, while resizing

**Reply by oleh-andriiv** [reply]
So, it's about moving across the rows by Y axis, not horizontally

**Reply by samuelarbibe** [reply, marked as answer]
Oh! I now understand.
This behavior would require you to create a custom resizing mechanism.  
The row/column relation in the provided example is pretty unconventional (both axis describing time on different scales, the y axis is basically a wrapping of the x axis and not a separate resource), so it will require a pretty advanced implementation. Possible, but not worth it in my opinion.

I Would suggest finding a different choice of axis selection, where the x and y axis describe a different set of values.
